cmd/promu.go

```diff
@@ -156,3 +156,12 @@ func envOr(name, def string) string {
 	}
 	return s
 }
+
+func checkRequiredConfigurations(configVars []string) {
+	for _, configVar := range configVars {
+		if !viper.IsSet(configVar) {
+			err := fmt.Errorf("'%s'", configVar)
+			fatalMsg(err, "Missing required configuration")
```

Comment:
Nope no reason. `fatalMsg(err error, msg string)` was a enhanced version of `fatal(err error)`.
I can change it in order to remain idiomatic to Go.

Moreover, I should have used `fatal()` with the full message as a `Error`.